editor/src/utils/utils.test-utils.tsx

```diff
@@ -575,3 +575,15 @@ export function boundingClientRectToCanvasRectangle(
 
   return canvasBounds
 }
+
+export async function searchInFloatingMenu(editor: EditorRenderResult, query: string) {
+  const floatingMenu = editor.renderedDOM.getByTestId(CanvasToolbarSearchTestID)
+  const searchBox = queryByAttribute('type', floatingMenu, 'text')!
+
+  await act(() => {
+    fireEvent.focus(searchBox)
+    fireEvent.change(searchBox, { target: { value: query } })
+    fireEvent.blur(searchBox)
```

Comment:
I'm not sure if it makes any difference here, but I'd normally expect `blur` to trigger after `keydown`.